api/services/managedInstance/aws/schemas/mi-compute.yaml

```diff
@@ -39,14 +39,23 @@ properties:
     properties:
       instanceTypes:
         type: object
-        required:
-          - preferredType
         properties:
           preferredType:
             type: string
             example: t2.micro
             description: >
-              Comma separated list of available instance types for instance
+              Prioritize a single instance type. The instance type must be included in the selected instance types or 
+              meet the resource requirements if defined. Note: Either preferredType or preferredTypes can be defined, 
+              but not both.
+          preferredTypes:
+            type: array
+            description: |
```

Comment:
What's the difference between "description: |" and "description: >" that is used above?